hawk/app/views/layouts/application.html.haml

```diff
@@ -7,17 +7,17 @@
     .navbar.navbar-fixed-top.navbar-inverse{ role: "navigation" }
       .container-fluid{ style: "white-space: nobreak;" }
         .navbar-header
-          = link_to image_tag("head/white.png", size: "160x23"), root_path, class: "navbar-brand"
+          = link_to image_tag("head/white.png", size: "120x40"), root_path, class: "navbar-brand"
 
         #navbar.navbar-right
           = render partial: "shared/usernav"
 
     .container-fluid
       .row.row-offcanvas.row-offcanvas-left
-        #sidebar.col-xs-6.col-sm-4.col-md-4.col-lg-2.sidebar-offcanvas
+        #sidebar.col-xs-6.col-sm-2.col-md-4.col-lg-2.sidebar-offcanvas
```

Comment:
I resized the browser page and it looks fine,
I did not tested on smartphone, but I don't thing we have to support those devides